main/docs/customize/hooks/view-logs-for-hooks.mdx

```diff
@@ -0,0 +1,38 @@
+---
+'og:description': Learn how to view logs for hooks using the Auth0 Dashboard.
+'og:image': https://cdn2.auth0.com/docs/1.14553.0/img/share-image.png

```

Comment:
Not urgent: we need to change the asset url to `docs-v2's` CDN url to remove the dependency from legacy docs.